workspaces/adr/.changeset/healthy-horses-wait.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage-community/plugin-adr-backend': patch
```

Comment:
Since this still a pre-major version package, feel free to make this a breaking change (minor bump) and go ahead and just remove these types from this package - just make sure to indicate it here with a `BREAKING CHANGE` note

Reply:
and also don't forget to remove related deps from this package that are no longer necessary

Reply:
right currently it is a patch but I think its ok to actually make this a breaking minor bump since this plugin is pre major v1 so there's no obligation to provide a deprecation period

Reply:
Let me make that change then. I'll just remove the dependencies from the backend package in favor of the module instead, and add a tiny breaking change message in the release notes